open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/processor/DataEngineProxyService.java

```diff
@@ -47,17 +50,22 @@ public void start() throws ConnectorCheckedException, UserNotAuthorizedException
         final String methodName = "start";
         this.auditLog.logMessage(methodName, DataEngineProxyAuditCode.INIT_POLLING.getMessageDefinition());
 
-
+        int pollIntervalInSeconds = 0;
```

Comment:
Looking better at this, it seems there is better way of controlling the same. 
I noticed [requiresPolling](https://github.com/odpi/egeria/blob/6d3cad4f5ed67eb9cd4cfa937febb1785924cabd/open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-connector/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/connectors/DataEngineConnectorBase.java#L39) in the connector interface.
Lets do the initialization based on this connector implementation property (we keep it true by default as is for the current implementation and we override it to return false for the new connector).

I think this is cleaner solution and we do not have to explain the special uses of "pollIntervalInSeconds" property in the documentation.

Reply:
updated